CONTRIBUTING.md

```diff
@@ -213,6 +218,34 @@ and will not work on this URL:
 
 [https://michigan-benefits-staging.herokuapp.com](https://michigan-benefits-staging.herokuapp.com)
 
+### Tracker flow and integrations
```

Comment:
this section may be more useful above, since I didn't read it until afterr I'd read the stuff about accept/reject